src/components/shared/Description.tsx

```diff
@@ -1,18 +1,19 @@
 import { MarkdownViewer } from '@stoplight/markdown-viewer';
 import { Popover } from '@stoplight/ui-kit';
+import { Box } from '@stoplight/mosaic';
 import * as React from 'react';
 
 export const Description: React.FunctionComponent<{ value: string }> = ({ value }) => (
   <Popover
     boundary="window"
     interactionKind="hover"
-    className="ml-2 flex-1 truncate flex items-baseline"
-    target={<div className="text-darken-7 dark:text-lighten-7 w-full truncate">{value}</div>}
+    className="flex-1 truncate flex items-baseline"
+    target={<Box className="text-darken-7 dark:text-lighten-7 w-full truncate">{value}</Box>}
```

Comment:
Somethig like `<Box textOverflow="truncate" w="full" color="muted"></Box>` - not 100% sure on the color without looking at rendered version - might be a better option.